cmake/external_protobuf.cmake

```diff
@@ -21,14 +21,15 @@ include(ExternalProject)
 # Download and install Google Protobuf ...
 #------------------------------------------------------------------------------
 
-set(PROTOBUF_GIT_REPO_URL https://github.com/google/protobuf.git)
-set(PROTOBUF_GIT_BRANCH origin/3.5.x)
+# This version of PROTOBUF is required by Microsoft ONNX Runtime.
+set(NGRAPH_PROTOBUF_GIT_REPO_URL "https://github.com/google/protobuf.git")
```

Comment:
Repo name as changed to "https://github.com/protocolbuffers/protobuf" Ri-direction happens correctly but would be better to use the updated official URL here.